Scripts/Modules/HierarchyModule.cs

```diff
@@ -60,141 +79,143 @@ List<string> GetFilterList()
 
 		List<HierarchyData> GetHierarchyData()
 		{
-			if (m_HierarchyData == null)
-				return new List<HierarchyData>();
-
-			return m_HierarchyData.children;
+			return m_HierarchyData ?? new List<HierarchyData>();
```

Comment:
Actually, we initialize it by default so it should never be null. We also try to re-use the same list as much as possible, so this could actually just turn into return m_HierarchyData;